shai/src/main/scala/at/forsyte/apalache/shai/v1/RpcServer.scala

```diff
@@ -39,6 +39,13 @@ class RpcServer(override val port: Int) extends ServerMain with LazyLogging {
   // https://github.com/informalsystems/apalache/issues/2622
   override def builder: (x0) forSome { type x0 <: ServerBuilder[x0] } =
     super.builder.maxInboundMessageSize(8 * 1024 * 1024)
+
+  override def run(args: List[String]) = {
+    myAppLogic.catchAll { t =>
+      logger.error("Error while starting Apalache server", t)
```

Comment:
Could we check the error, and if it is a socket binding problem, just report that without logging the traceback? Otherwise, reporting the full traceback would be helpful, since the failure is for an unknown reason.

Reply:
Will try